README.adoc

```diff
@@ -42,4 +42,20 @@ This repository contains:
 - TBTC v2 link:docs/[documentation],
 - TBTC v2 Yearn vault link:yearn/[Solidity smart contracts].
 
-toc::[]
\ No newline at end of file
+toc::[]
+
+== Overview
+tBTCv2 uses the
+link:https://github.com/keep-network/keep-core/tree/main/solidity/random-beacon[Random
+Beacon] and link:https://github.com/keep-network/sortition-pools[Sortition Pool]
+to generate governable-sized (starting at 51-of-100)
+link:https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm[ECDSA-backed]
```

Comment:
Maybe threshold ECDSA and link to https://eprint.iacr.org/2019/114.pdf ?

Reply:
[3bde8ca](https://github.com/keep-network/tbtc-v2/pull/323/commits/3bde8ca2d7f761ef9e79a385354d31a63083dbfa)